package.json

```diff
@@ -98,7 +98,7 @@
     "react-redux": "6.0.0",
     "react-test-renderer": "16.7.0",
     "redux": "4.0.1",
-    "redux-saga": "0.16.2",
+    "redux-saga": "1.0.1",
```

Comment:
Do not merge this dependency update. It would break the API!
See [this issue](https://github.com/redux-saga/redux-saga/issues/1743) for more information.